src/python/tests/genome_metadata/test_dump.py

```diff
@@ -182,13 +302,129 @@ def test_get_genome_metadata(
 
     Args:
         mock_session: A mock of `sqlalchemy.orm.Session()` class.
+        db_name:
         meta_data: `meta` table content in a list of named tuples.
         output: Expected genome metadata dictionary.
         expectation: Context manager for the expected exception (if any).
     """
+    # pylint: disable=too-many-positional-arguments
```

Comment:
I would actually add this at the top of the file, so every current and future method will not raise this warning.